Warden's whistle to replace warden's cap steal objective

arenaconspiracy opened the pull request:
## About the PR
Adds new warden's whistle item, guaranteed to spawn in the warden's locker. Replaces thief warden's cap objective with whistle.

## Why / Balance
The warden's cap is an uninteresting, trivial steal objective. It is not guaranteed to be mapped, and may be acquired from the Head of Personnel's uniform printer. The whistle is guaranteed to exist on any map with a warden's locker (all of them), and is unique, necessitating stealing it from the warden's person if they are wearing it.

## Technical details
Prototype changes in yaml. New sprites with matching metadata. New sound group, using sounds recorded by zank lofi (asheviolet) and modified by me.

## Media
https://github.com/user-attachments/assets/c2adbe9e-1cb9-428b-a862-6b916200e294

## Requirements
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

**Changelog**
:cl:
- add: The warden's locker now starts with a unique whistle
- tweak: The thief objective for the warden's cap has been replaced with the warden's whistle

imatsoup commented on Resources/Audio/Items/Whistle/attributions.yml:
```diff
@@ -17,3 +17,11 @@
+  - "warden_whistle_3.ogg"
+  license: "CC0-1.0"
+  copyright: "Recorded by zank lofi. Modified by arenaconspiracy on github"
+  source: "N/A"
```
Causing a yaml linter check to fail, as it expects a url here.

arenaconspiracy replied:
I am aware. I have been waiting on a response from the person who recorded the sounds I processed, on what link they would like in attributions.

bruhmogus commented:
gonna be real this just kinda... nukes the only steal objective that isn't "thief glove it and hope you dont get caught" or "drag it to your beacon and pray to god nobody sees you"

arenaconspiracy commented:
Added licensing comments in accordance to https://github.com/RonRonstation/ronstation/issues/296 and refactored warden's lockers to use reusable entity tables instead of a set fill.
